ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -12,31 +12,24 @@
 
 import com.codeborne.selenide.Selenide;
 
-import java.util.HashMap;
-
 import cucumber.api.java.en.Then;
 
 public class DocsVerificationSteps {
-    private HashMap<String, String> versions = new HashMap<>();
-    private static String versionOnAboutPage;
-
-    public DocsVerificationSteps() {
-        versions.put("1.9", "7.6");
-    }
 
     @Then("^check version in about page$")
     public void checkVersion() {
-        By syndesisVersion = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
-        assertThat($(syndesisVersion).shouldBe(visible).getText())
+        By versionOnAboutPage = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
+        assertThat($(versionOnAboutPage).shouldBe(visible).getText())
             .isNotEmpty();
-        versionOnAboutPage = $(syndesisVersion).getText().substring(0, 3);
+        assertThat($(versionOnAboutPage).getText()).isEqualTo(TestUtils.getSyndesisVersion());
+        assertThat($(versionOnAboutPage).getText().substring(0, 4)).isEqualTo("1.10");
```

Comment:
it would be better for you to create a constant in the class for these values, so that you don't need to search it in the code when you will want to change it